crates/common/flockfile/src/unix.rs

```diff
@@ -173,6 +172,7 @@ mod tests {
 
     use super::*;
     use assert_matches::*;
+    use nix::unistd::Pid;
```

Comment:
Why not also use `std::process::id()` in the tests?

Reply:
Thought of having a different way of getting the `pid` for testing purposes.